tests/mountingOptions/stubs.global.spec.ts

```diff
@@ -119,7 +119,7 @@ describe('mounting options: stubs', () => {
 
     expect(wrapper.html()).toEqual(
       '<div>\n' +
-        '  <foo-stub class="bar" test-id="foo" dynamic="[object Object]"></foo-stub>\n' +
+        '  <foo-stub dynamic="[object Object]" class="bar" test-id="foo"></foo-stub>\n' +
```

Comment:
Instead of fixing this issue... I wasn't able order of props and attributes.
If we need to keep this order... I'll try it !!

Reply:
The order of attributes has no specific meaning, so it's fine if it changes a bit. This is one of the things I dislike about snapshots; they often change, even if the component works the same.

Reply:
I got it !!
Thanks a lot !!